docs/ai-gateway.md

```
@@ -0,0 +1,49 @@
+```bash
+wrangler secret put CF_AIG_TOKEN --config worker/wrangler.toml
+wrangler secret put OPENAI_TOKEN --config worker/wrangler.toml
+```
+
+For local development, copy `dev.vars.example` to `.dev.vars` and fill in your values.
+
+## Gateway URL format
+
+The AI Gateway base URL is constructed as:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)